os/kernel/Kconfig

```diff
@@ -999,6 +999,7 @@ endmenu # Stack size information
 
 config LOG_DUMP
 	bool "Enable log dump functionality"
+	depends on LIBLZMA
```

Comment:
As you know, current TizenRT supports two types of compression, `LZMA` and `MINIZ`.
Can the log dump be worked only with `LZMA`?

Reply:
We can use LZMA only in log dump for now. But it's not our final goal. Let's just enable LZMA without dependency.

Reply:
done